templates/commands/goldentest/test_funcs.go

```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/abcxyz/abc/templates/common"
 	"github.com/abcxyz/abc/templates/common/render"
 	"github.com/abcxyz/abc/templates/model/decode"
-	goldentest "github.com/abcxyz/abc/templates/model/goldentest/v1alpha1"
+	goldentest "github.com/abcxyz/abc/templates/model/goldentest/v1beta3"
```

Comment:
Curious to learn the version management in abc repo.  Looks like in `templates/model/goldentest`, we maintain different versions of goldentest for example v1alpha1 and v1beta3. But in `templates/commands/goldentest`, we use the latest version which is v1beta3. In that case, who will be the user of `templates/model/goldentest/v1alpha1`, I guess the users with older cli versions? Does the command `cli` has its own version?

Reply:
The docs about api_version changes are a little rudimentary, but they're in templates/model/README.md (this was also linked in the PR description)

The fundamental design is that old `api_version`s are supported, they're just auto-upgraded to the latest api_version. The abc CLI app logic is written to use only the latest api_version. See the upgrade.go files and the `DecodeValidateUpgrade()` function to understand how api_version upgrading works.

Reply:
Thanks for explanation!